src/features/export-package-resources-modal/constants.js

```diff
@@ -130,6 +130,10 @@ const RESOURCE_EXPORT_FIELDS = [
 ];
 
 const PACKAGE_EXPORT_FIELDS = [
+  {
```

Comment:
Please also add to `RESOURCE_EXPORT_FIELDS` constants to export this from Package too

Reply:
Is it necessary to be able to select a `Provider Level Token` not only in the `Package fields to export` but also in the `Title fields to export`?

Reply:
Yes, it's in the ticket's requirements
> Given I click the action Export on a package or title+package (aka resource)